azure-devops/create-new-agent-image.ps1

```diff
@@ -278,8 +298,7 @@ $Vmss = Set-AzVmssStorageProfile `
   -VirtualMachineScaleSet $Vmss `
   -OsDiskCreateOption 'FromImage' `
   -OsDiskCaching ReadWrite `
-  -ImageReferenceId $Image.Id `
-  -ManagedDisk Premium_LRS

```

Comment:
Why is `-ManagedDisk Premium_LRS` being removed?

Reply:
@BillyONeal

Reply:
Because in the vcpkg version of this script they have both premium and non-premium disks in their image. The STL still gets premium with this change because that's how the image is minted.